CONTRIBUTING.md

```diff
@@ -122,19 +144,18 @@ See guidelines for [contribution with CSS](https://github.com/liferay/clay/blob/
 
 ### Documentation Style Guide
 
--   Use [JSDOC](http://usejsdoc.org/) for documentation in your JavaScript code.
-
-#### Example
-
-```javascript
-/**
- * Data to add to the element.
- * @default undefined
- * @instance
- * @memberof ClayActionsDropdown
- * @type {?object}
- */
-data: Config.object();
+Clay uses [TypeScript](https://www.typescriptlang.org/) for typed, thereby overwriting [JSDOC](http://usejsdoc.org/) writing but adding descriptions to `interface`, `type`, `enum`, etc... to make it easier to use components.
+
+```typescript
+interface IProps {
+	/**
+	 * Set to true means that network status information will be passed
+	 * via `renders props` and will also cause new renderings as
+	 * networkStatus changes, when false rendering does not
+	 * happen again.
```

Comment:
The English here is a bit weird so I'd try to pick another example. For example, maybe a nice short one from clay-drop-down?

```javascript
	/**
	 * Default position of menu element. Values come from `metal-position`.
	 */
	alignmentPosition?: number;
```